app/environments/brassbirmingham/brassbirmingham/envs/classes/board.py

```diff
@@ -1,33 +1,33 @@
 from __future__ import annotations
 
-from typing import TYPE_CHECKING, List
+from typing import TYPE_CHECKING, Dict, List
 
 from consts import (
     CANAL_PRICE,
+    MAX_MARKET_COAL,
+    MAX_MARKET_IRON,
     ONE_RAILROAD_COAL_PRICE,
     ONE_RAILROAD_PRICE,
     ROAD_LOCATIONS,
     STARTING_CARDS,
-    STARTING_WILD_BUILDING_CARDS,
```

Comment:
Are we **absolutely** sure there's no limit to these wild cards?

Reply:
Yep. The number of each wild card is equal to the number of players playing and no one can take more than 1 of each at once